composer.json

```diff
@@ -4,7 +4,8 @@
     "description": "Basic API to handle HTTP redirects with the Flow Framework",
     "license": "MIT",
     "require": {
-        "neos/flow": "^7.0 || ^8.0 || dev-master",
+        "php": ">=8.2",
```

Comment:
isnt this implicit set by neos/flow?

Reply:
Yes, that's true. But why shouldn't define this package his own dependencies, too?

Reply:
setting per package is IMHO good, we should really have it in every package and run static tests against the lowest version that is allowed.